src/roiextractors/extractors/femtonicsimagingextractor/femtonicsimagingextractor.py

```diff
@@ -0,0 +1,343 @@
+"""A Femtonics imaging extractor with simplified initialization."""
+
+from pathlib import Path
+from typing import Optional, Tuple, Dict, Any, List
+from warnings import warn
+from datetime import datetime, timezone
+import xml.etree.ElementTree as ET
+
+import numpy as np
+import h5py
+from lazy_ops import DatasetView
+
+from ...imagingextractor import ImagingExtractor
+from ...extraction_tools import PathType, FloatType, ArrayType
+
+
+class FemtonicsImagingExtractor(ImagingExtractor):
+    """A Femtonics imaging extractor with simplified initialization."""
+
+    extractor_name = "FemtonicsImaging"
+    is_writable = False
```

Comment:
I think `is_writable` is not needed anymore:

https://github.com/catalystneuro/roiextractors/pull/403